src/main/resources/public/ts/sniplets/content/workspace-nextcloud-content.sniplet.ts

```diff
@@ -145,7 +145,6 @@ class ViewModel implements IViewModel {
             if (folderContent.folder instanceof models.Element) {
                 let fileToMove: Set<SyncDocument> = new Set(this.selectedDocuments).add(document);
                 let finalUpload: Array<string> = Array.from(fileToMove)
-                    .filter((file: SyncDocument) => !file.isFolder)
```

Comment:
?

Reply:
Before the last update, moving folders was not handled by the API. To avoid useless call to an API who does not handle folders, the front was only sending the request when it was needed. Now the API can handle folder move, so we removed the filter and the front now call the API for folders and files.